subworkflows/ebi-metagenomics/rrna_extraction/main.nf

```diff
@@ -0,0 +1,42 @@
+
+include { INFERNAL_CMSEARCH           } from '../../../modules/ebi-metagenomics/infernal/cmsearch/main'
+include { CMSEARCHTBLOUTDEOVERLAP     } from '../../../modules/ebi-metagenomics/cmsearchtbloutdeoverlap/main'
+include { EASEL_ESLSFETCH             } from '../../../modules/ebi-metagenomics/easel/eslsfetch/main'
+
+workflow RRNA_EXTRACTION {
+
+    take:
+    ch_fasta // channel: [ val(meta), [ fasta ] ]
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    INFERNAL_CMSEARCH(
+        ch_fasta,
+        file(params.rfam)
```

Comment:
Is rfam a file or folder with models? (probably it would be better to add that to name like `rfam_model_file`/`rfam_model_dir`)
I'm wondering how good is to take db directly from params. We can re-use the same subwf in one pipeline with various sets of rfams..

Reply:
rfam is a file, but good idea for folders future modules!

I think we can still take it from params even if it's used more than once because you could just add another param for each rfam/database.. do you have a suggestion for a better way though?

Reply:
done!